frontend/apps/app/app/(app)/app/projects/[projectId]/ref/[branchOrCommit]/knowledge-suggestions/page.tsx

```diff
@@ -1,17 +1,24 @@
 import type { PageProps } from '@/app/types'
 import { KnowledgeSuggestionsListPage } from '@/features/projects/pages/KnowledgeSuggestionsListPage'
+import { branchOrCommitSchema } from '@/utils/routes'
 import { notFound } from 'next/navigation'
 import * as v from 'valibot'
 
 const paramsSchema = v.object({
   projectId: v.string(),
+  branchOrCommit: branchOrCommitSchema,
 })
 
 export default async function KnowledgeSuggestionsPage({ params }: PageProps) {
-  const parsedParams = v.safeParse(paramsSchema, await params)
+  const parsedParams = v.safeParse(paramsSchema, params)
```

Comment:
Q: Why are you delete `await`? Perhaps it is necessary.

Reply:
Thank you for noticing. I had accidentally deleted it.I have corrected it.🙏
https://github.com/liam-hq/liam/pull/1052/commits/4bb7840aec53792fa0095dfd71f036a2f012dc3e